src/components/ContentCard/ContentCard.stories.tsx

```diff
@@ -0,0 +1,53 @@
+import type { Meta, StoryObj } from '@storybook/react';
+
+import ContentCard from './ContentCard';
+import Tag from '../Tag/Tag';
+import Image from '../Image/Image';
+
+const meta = {
+  title: 'Components/ContentCard',
+  component: ContentCard,
+  parameters: {
+    layout: 'centered'
```

Comment:
we may want to remove this since the non-centered layout will be truer to the actual behavior of card, a la accordion